crates/astria-composer/src/searcher/geth_collector.rs

```diff
@@ -1,3 +1,14 @@
+//! `GethCollector` is responsible for fetching pending transactions submitted to rollup
```

Comment:
Make the first sentence in the module shorter. It's used as a summary when listing the module (we are not generating docs because this module is private, but it's still good practice).

```suggestion
//! Fetch transactions from a Geth rollup node.
```